_episodes/05-comparing-agile-methodologies.md

```diff
@@ -0,0 +1,109 @@
+> > | Cadance             | Regular fixed length sprints (i.e. 2 weeks) | Continuous flow |
+> > | Release Methodology | At the end of each sprint | Continuous delivery      |
+> > | Roles               | Product owner, scrum master, development team | No required roles |
+> > | Key Metrics         | Velocity | Lead time, cycle time, WIP |
+> > | Change Philosophy   | Teams should not make changes during the sprint | Change can happen at any time |
+> >
+> > NB: Taken from https://www.atlassian.com/agile/kanban/kanban-vs-scrum
+> {: .solution}
+{: .challenge}
+
+# Scrum and Kanban are both Agile methodologies
+Scrum and Kanban are both Agile methodologies.
+They both provide a set of practical tools that allow you to work according to the Agile manifesto and principles.
+
+> ## Exercise: Compare Scrum and Kanban with the Agile manifesto principles
+> Pick one or more principles from the [12 principles behind the agile manifesto](http://agilemanifesto.org/principles.html)
+> that we discussed in episode 2.
+> **Write down how Scrum and Kanban practically implement these principles.**
+> 1. Our highest priority is to satisfy the customer through early and continuous delivery of valuable software.
+> 2. Welcome changing requirements, even late in development. Agile processes harness change for the customer's competitive advantage.
+> 3. Deliver working software frequently, from a couple of weeks to a couple of months, with a preference to the shorter timescale.
+> 4. Business people and developers must work together daily throughout the project.
+> 5. Build projects around motivated individuals. Give them the environment and support they need, and trust them to get the job done.
+> 6. The most efficient and effective method of conveying information to and within a development team is face-to-face conversation.
+> 7. Working software is the primary measure of progress.
+> 8. Agile processes promote sustainable development. The sponsors, developers, and users should be able to maintain a constant pace indefinitely.
+> 9. Continuous attention to technical excellence and good design enhances agility.
+> 10. Simplicity--the art of maximizing the amount of work not done--is essential.
+> 11. The best architectures, requirements, and designs emerge from self-organizing teams.
+> 12. At regular intervals, the team reflects on how to become more effective, then tunes and adjusts its behavior accordingly.
+>
+> > ## Solution
+> > Example solution for principle 3: Deliver working software frequently, from a couple of weeks to a couple of months,
+> > with a preference to the shorter timescale.
+> > - **Kanban**: Continuous delivery (release after every finished item)
+> > - **Scrum**: Release after every sprint (for example every 2 weeks)
+> {: .solution}
+{: .challenge}
+
+# ScrumBan and hybrid methodologies
+It is very common to do a mixture of Scrum and Kanban, picking the best of both worlds according to the team's needs.
```

Comment:
Yes, that is indeed a very good point. (Although I would personally disagree, especially for RSEs, but it's better to not teach my opinion :))

Reply:
I also personally disagree with the statement that not adopting the full scrum framework is bound to fail, I think that this orthodoxy is in itself not very agile and in conflict with the 4th value of the agile manifesto: Responding to change over following a plan. It is of course okay to avoid calling your method scrum if you cherry picked some items from scrum, just for clarity.